azurerm/resource_arm_virtual_machine_scale_set.go

```diff
@@ -379,6 +379,13 @@ func resourceArmVirtualMachineScaleSet() *schema.Resource {
 										Set:      schema.HashString,
 									},
 
+									"application_security_group_ids": {
+										Type:     schema.TypeSet,
+										Optional: true,
```

Comment:
is there a limit on the number of Application Security Groups which a VM can be connected to? I have a feeling this may be 1, but I've not checked recently?

Reply:
It looks like from the networking limits page that this is probably limited to 20 Application Security Groups. 
`Application security groups per IP configuration, per NIC | 20`

Reply:
As these limits can change, I assume that it is best to leave this validation to the Azure API?